src/components/list/list.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { ListItem } from '../listItems/listItem';
+import './list.scss';
+
+interface Prop {
+  obj: Array<Array<number | string>>;
```

Comment:
be more verbose obj is not good  to describe what it is


if is and Array I sugguest `items`